internal/job/backup/backup.go

```diff
@@ -35,6 +38,44 @@ type Backup struct {
 	targetPVCUID              string
 	maxPartSize               uint64
 	expansionUnitSize         uint64
+	diffChecksumChunkSize     uint64
+}
+
+func WriteDiffPart(stream io.ReadCloser, outputPath, checksumPath string, chunkSize uint64) (err error) {
```

Comment:
Could you add a new argument that toggles whether replacing `writer` with csum writer or not? In addition, please write a new test to verify csum file will/won't be created if this flag is true/false.